scss/components/_article.scss

```diff
@@ -0,0 +1,223 @@
+.article{
+    padding: 0 20px;
+    #article-container{
+        & > div:first-child{
+            max-width: 1200px;
+            margin: 0 auto;
+            padding: 60px 0px 190px;
+            display: flex;
+            flex-direction: column;
+            @include md-screen{
+                padding: 45px 0 70px;
+            }
+            .date{
+                p {
+                    color: #383F43;
+                }
+            }
+            .article-top{
+                #article-top-container{
+                    & > div:first-child{
+                        display: flex;
+                        justify-content: space-between;
+                        @include md-screen{
+                            flex-direction: column;
+                            margin-bottom: 80px;
+                        }
+                        .heading-author {
```

Comment:
There is a LOT of unnecessary nesting of style rules in this file which creates a bloated compiled file. This line compiles to `.article #article-container > div:first-child .article-top #article-top-container > div:first-child .heading-author {}` when it could probably just be `.article .heading-author {}`. Please reevaluate all style rule nesting in every SCSS file.

Reply:
And don't forget that elements inherit styles from their parents. You don't always have to target the `<h1>` or `<p>` directly.

Reply:
@Prystal254 Don't forget about this.